client/tracks/index.js

```diff
@@ -14,7 +14,9 @@ function isEnabled() {
  * @param {Object?} eventProperties Event properties.
  */
 function recordEvent( eventName, eventProperties ) {
-	window.wcTracks.recordEvent( eventName, eventProperties );
+	const recordFunction =
+		window.wc.tracks.recordEvent ?? window.wcTracks.recordEvent;
```

Comment:
Do you know WC version when `wc.tracks.recordEvent` was introduced? I'm afraid it can throw if e.g. `window.wc.tracks` is null.

Reply:
Thanks for catching this! I didn't find out which WC version it was introduced, but I've gone ahead and added optional chaining to it so it doesn't throw in ce02d27.

I've tested this with WC 4.0 (wcpay's minimum WC supported version), and it works.